src/Language/PureScript/Errors.hs

```diff
@@ -514,32 +559,37 @@ prettyPrintSingleError full e = prettyPrintErrorMessage (if full then e else sim
     unwrap pos (NotYetDefined ns err) = NotYetDefined ns (unwrap pos err)
     unwrap _   (PositionedError pos err) = unwrap (Just pos) err
     unwrap pos other = wrap pos other
-  
+
     wrap :: Maybe SourceSpan -> ErrorMessage -> ErrorMessage
     wrap Nothing    = id
     wrap (Just pos) = PositionedError pos
 
+
 -- |
 -- Pretty print multiple errors
 --
 prettyPrintMultipleErrors :: Bool -> MultipleErrors -> String
-prettyPrintMultipleErrors = prettyPrintMultipleErrorsWith "Error:" "Multiple errors:"
-    
+prettyPrintMultipleErrors = curry $ flip evalState [] . (uncurry $ prettyPrintMultipleErrorsWith "Error:" "Multiple errors:")
```

Comment:
I find this pointfree-style code a little hard to read. Can we just keep the `full` argument as a named argument?